api/classes/publisher.js

```diff
@@ -78,7 +78,7 @@ function upload(path, buffer, remixMetadata) {
   }
   var headers = {
     'Cache-Control': 'max-age=0',
-    'Content-Type': mimeType,
+    'Content-Type': mimeType + '; charset=utf-8',
```

Comment:
w00p, never mind, http://tools.ietf.org/html/rfc2616#section-3.4 tells me it's case insensitive =P